tests/golden_tests.rs

```diff
@@ -95,15 +95,15 @@ fn compile_term() {
 fn compile_file() {
```

Comment:
Can we swap the names of these tests? Make this one `compile_file_o_all` and the other one just `compile_file`